src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/UserController.java

```diff
@@ -126,9 +126,8 @@ public ResponseEntity<String> updateFundingOrganisation(@RequestBody UpdateFundi
             userService.updateFundingOrganisation(grantAdmin.get(), updateFundingOrgDto.departmentName());
             return ResponseEntity.ok("User's funding organisation updated successfully");
         }
-        else {
-            return ResponseEntity.status(404).body("No grant Admin found with sub " + jwtPayload.getSub());
-        }
+        return ResponseEntity.ok(
```

Comment:
Any reason we are not handling the 404 upstream? seems sensible to throw 404 when the admin doesn't exist.